codec/src/main/java/io/netty/handler/codec/ConvertibleHeaders.java

```diff
@@ -90,13 +90,6 @@
     List<ConvertedType> getAllAndRemoveAndConvert(UnconvertedType name);
 
     /**
-     * Invokes {@link Headers#entries()} and lazily does a conversion on the results as they are accessed
```

Comment:
@Scottmitch yeah getting rid of `ConvertibleHeaders` would be great if possible. Could we provide it as utility methods for `Headers` instead?